addons/mail/static/tests/tours/mail_composer_test_tour.js

```diff
@@ -124,5 +124,30 @@ registry.category("web_tour.tours").add("mail/static/tests/tours/mail_composer_t
             trigger: '.o-mail-Message .o-mail-AttachmentCard:contains("text.txt")',
             isCheck: true,
         },
+        // Test the full composer input text is kept on closing
+        {
+            content: "Open full composer",
+            trigger: "button[aria-label='Full composer']",
+        },
+        {
+            content: "Write something in full composer",
+            trigger: "#body_0",
+            run: "text keep the content",
+        },
+        {
+            content: "Discard full composer",
+            trigger: ".btn-close",
+        },
```

Comment:
"Discard" =/= "Close". The nuance is important because the former implies clearing the content while the latter doesn't. Here it should be "Close" rather than "Discard".